src/FixedDataTable.js

```diff
@@ -1025,6 +1025,7 @@ var FixedDataTable = createReactClass({
     var totalHeightReserved = props.footerHeight + props.headerHeight +
         groupHeaderHeight + 2 * BORDER_HEIGHT;
     var bodyHeight = height - totalHeightReserved;
+
     var scrollContentHeight = this._scrollHelper.getContentHeight();
     var totalHeightNeeded = scrollContentHeight + totalHeightReserved;
```

Comment:
Can you delete this line and move the var declaration down below?

Afterwards ping me & I'll fix & release